src/core/include/openvino/core/validation_util.hpp

```diff
@@ -113,7 +113,7 @@ int64_t normalize_axis(const std::string& node_description,
 /// \param[in]  axes              The requested vector of axes.
 ///
 OPENVINO_API
-void normalize_axes(const Node* node, const int64_t& tensor_rank, std::vector<int64_t>& axes);
+void normalize_axes(const Node* node, int64_t tensor_rank, std::vector<int64_t>& axes);
```

Comment:
can I ask why this is changed to non const type? seems there is no reason. If there is no critical reason, could you please revert the unnecessary change?